packages/generator/src/prisma/field.ts

```diff
@@ -56,7 +56,7 @@ export class Field {
 
   // 'name:type?[]:attribute' => Field
   static parse(input: string): Field[] {
-    const [_fieldName, _fieldType, attribute] = input.split(':')
+    const [_fieldName, _fieldType = 'String', attribute] = input.split(':')
```

Comment:
just a heads up @flybayer, this deafult is already set down in the `Field` constructor rather than in `parse` so that directly constructed instances get this benefit, I think we can remove this

Reply:
Oh.. I didn't realize because it wasn't working. `_fieldType` must be set here because it's used just two lines below I believe.